public/js/Keys.js

```diff
@@ -58,7 +58,7 @@ class Keys {
 
 	isPlayerOnMouse() {
 		return Math.abs(this._localPlayer.getX() * Globals.widthRatio - this._mouseX) < 15
-		&& Math.abs(this._localPlayer.getY() * Globals.heightRatio - this._mouseY) < 15;
+			&& Math.abs(this._localPlayer.getY() * Globals.heightRatio - this._mouseY) < 15;
```

Comment:
Hmm, check out your tabs... the settings should be converting tabs to spaces and set it to 2 spaces